packages/openactive-integration-tests/test/helpers/request-state.js

```diff
@@ -87,7 +88,7 @@ class RequestState {
      */
     this.testInterfaceResponses = await Promise.all((this.orderItemCriteriaList || []).map(async (orderItemCriteriaItem, i) => {
       // If an opportunity is available for reuse, return it
-      if (orderItemCriteriaItem.hasOwnProperty('opportunityReuseKey') && reusableOpportunityPromises.has(orderItemCriteriaItem.opportunityReuseKey)) {
+      if (orderItemCriteriaItem.hasOwnProperty('opportunityReuseKey') && orderItemCriteriaItem.opportunityReuseKey !== null && reusableOpportunityPromises.has(orderItemCriteriaItem.opportunityReuseKey)) {
```

Comment:
```suggestion
      if (!_.isNil(orderItemCriteriaItem.opportunityReuseKey') && reusableOpportunityPromises.has(orderItemCriteriaItem.opportunityReuseKey)) {
```

and lodash would need to be imported at the top

This is a bit more generic ("field not present", "field value is null or undefined" would all be caught by this) and doesn't use the `{}.hasOwnProperty()` method, which [can cause issues](https://eslint.org/docs/rules/no-prototype-builtins)